File: src/main/java/cloud/common/Restore.java
package cloud.common;

import se.sics.kompics.Event;
import se.sics.kompics.address.Address;

/**
 * 
 * @author Amir Moulavi
 * @date 2011-03-27
 *
 */

public class Restore extends Event {

	private Address node;

	public Restore(Address node) {
		this.node = node;
	}

	public Address getNode() {
		return node;
	}
	
}

File: src/main/java/cloud/common/Restore.scala
package cloud.common

import se.sics.kompics.Event
import se.sics.kompics.address.Address
import reflect.BeanProperty

/**
 * @author Amir Moulavi
 * @date 2011-07-09
 *
 */
class Restore(@BeanProperty var node:Address) extends Event